### DISCUSSION TITLE
[@types/node] v20.x typings

### DISCUSSION THREAD

**Discussion by Semigradsky** [DefinitelyTyped/DefinitelyTyped#65249] [Issues with a @types package] [answered]
Node.js 20.0.0 is out.
Isn't it time to create `v18` folder and start publishing v20 typings? Or do we want to wait until v20 becomes LTS (October)?

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
I believe the intent is to support 'current' (and folks are expecting this), just the reality check here with lack of resources, which is main limiting factor IMO.
thanks!

**Reply by Semigradsky** [reply]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65297
